applications/milliwatt/src/milliwatt.hrl

```diff
@@ -20,13 +20,11 @@
 
 -define(RESOURCE_TYPES_HANDLED, [<<"audio">>, <<"video">>]).
 
--define(TONE, kapps_config:get_non_empty(?CONFIG_CAT
-                                        ,<<"tone">>
-                                        ,kz_json:from_list(
-                                           [{<<"caller_id">>,[]}
-                                           ,{<<"number">>,[<<"5555555551">>]}
-                                           ]))
-       ).
+-define(DEFAULT_TONE, kz_json:from_list(
+                        [{<<"caller_id">>,[]}
+                        ,{<<"number">>,[<<"5555555551">>]}
+                        ])).
```

Comment:
The project drops the matching parens/brackets on these multi-line things onto the next line.